cmd/lncli/commands.go

```diff
@@ -2476,14 +2499,17 @@ var restoreChanBackupCommand = cli.Command{
 	channel. If successful, this command will allows the user to recover
 	the settled funds stored in the recovered channels.
 
-	The command will accept backups in one of three forms:
+	The command will accept backups in one of four forms:
 
 	   * A single channel packed SCB, which can be obtained from
 	     exportchanbackup. This should be passed in hex encoded format.
 
 	   * A packed multi-channel SCB, which couples several individual
 	     static channel backups in single blob.
 
+	   * A file path which points to a packed single-channel backup within a
+	     file, using the same format that lnd does in its channel.backup file.
```

Comment:
This line is longer than 80 characters (when using 8 spaces for the tab character in your editor), that's why the linter is complaining.